### PR TITLE
perf(ui/switch): add class

### PR DESCRIPTION
**Pull request by chunhuigao**
## Checklist

---

- [ ] Fix linting errors
- [ ] Tests have been added / updated (or snapshots)

## Change information

---

### CODE REVIEW

File: packages/varlet-ui/src/switch/Switch.vue [hunk 1]
```diff
@@ -1,33 +1,31 @@
 <template>
-  <div class="var-switch">
+  <div :class="n()">
     <div
-      class="var-switch-block"
-      :class="[disabled || formDisabled ? 'var-switch__disable' : null]"
+      :class="classes('var-switch-block', [disabled || formDisabled, n('disable')])"
```
Comment (BeADre) [inline packages/varlet-ui/src/switch/Switch.vue:4]: `var-switch-block`我命名的有问题，改成`var-switch__block`， 顺便 less 文件里面对应改一下

Comment (chunhuigao) [reply packages/varlet-ui/src/switch/Switch.vue:4]: ok

Comment (chunhuigao) [reply packages/varlet-ui/src/switch/Switch.vue:4]: ok

Comment (chunhuigao) [reply packages/varlet-ui/src/switch/Switch.vue:4]: Does the test file need to be modified?

Comment (BeADre) [reply packages/varlet-ui/src/switch/Switch.vue:4]: e.g?

Comment (chunhuigao) [reply packages/varlet-ui/src/switch/Switch.vue:4]: yarn test file

Comment (chunhuigao) [reply packages/varlet-ui/src/switch/Switch.vue]: 测试用例中的快照是var-switch-disable ，如果只是.vue和.less修改，yarn test 会失败

Comment (BeADre) [reply packages/varlet-ui/src/switch/Switch.vue]: 那单测就一起改

Comment (chunhuigao) [reply packages/varlet-ui/src/switch/Switch.vue]: 好的

**Review by BeADre — APPROVED** [review]

**Comment by BeADre** [comment]
thanks

**Pull request merged**